sql/catalyst/src/main/scala/org/apache/spark/sql/catalyst/parser/ASTNode.scala

```diff
@@ -22,6 +22,7 @@ import org.apache.spark.sql.catalyst.trees.{Origin, TreeNode}
 
 case class ASTNode(
     token: Token,
+    callSiteName: String,
```

Comment:
I don't think we need to add `callSiteName` because we can get call sites by `Utils.getCallSite`.